src/components/EmptySection.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="s-empty-section">
+    <i class="s-icon-search" v-if="this.variation === 'search'"></i>
+    <i class="s-icon-hide" v-if="this.variation === 'text'"></i>
+    <div class="s-empty-section--title">{{ title }}</div>
```

Comment:
for proper bem these would be
`s-empty-section__title`
`s-empty-section__subtitle`

__ = element
-- = variation